src/server/scripts/Outland/GruulsLair/boss_high_king_maulgar.cpp

```diff
@@ -112,12 +112,14 @@ class boss_high_king_maulgar : public CreatureScript
             {
                 Talk(SAY_OGRE_DEATH);
                 if (actionId == MAX_ADD_NUMBER)
-                    me->SetLootMode(1);
+                {
+                    me->AddLootMode(1);
+                }
             }
             else if (actionId == MAX_ADD_NUMBER)
             {
                 me->loot.clear();
```

Comment:
Maulgar needs lootmode 1 added manually, like the 1 which was removed.
```cpp
me->AddLootMode(1);
```

Reply:
I'm obviously doing this code suggestion thing wrong. I hope my intentions are still clear 😓